sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1776,6 +1776,24 @@ object functions {
   }
 
   /**
+   * Returns string, with the first letter of each word in uppercase,
```

Comment:
I think we should update the doc, should not turn other things to lower case, for example WTO should remain the same.